hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/ErrorTableUtils.java

```diff
@@ -28,32 +28,39 @@
 import org.apache.hudi.exception.HoodieException;
 
 import org.apache.hadoop.fs.FileSystem;
+import org.apache.hudi.exception.HoodieValidationException;
 import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
 import org.apache.spark.sql.SparkSession;
 
+import static org.apache.spark.sql.functions.lit;
+
 import java.lang.reflect.InvocationTargetException;
 import java.util.Arrays;
+import java.util.stream.Collectors;
 
 import static org.apache.hudi.config.HoodieErrorTableConfig.ERROR_TABLE_WRITE_CLASS;
 import static org.apache.hudi.config.HoodieErrorTableConfig.ERROR_TABLE_WRITE_FAILURE_STRATEGY;
+import static org.apache.hudi.utilities.deltastreamer.BaseErrorTableWriter.ERROR_TABLE_CURRUPT_RECORD_COL_NAME;
```

Comment:
So this is like a reserved column for any hudi table then? 
should we check if incoming dataset has this column and fail may be? what in case we have a data column w/ the same name? 
how happens incase of incremental consumer? it might have this column as a data column right ?

Reply:
`ERROR_TABLE_CURRUPT_RECORD_COL_NAME` currently this is fixed column . which is same as spark default `_corrupt_record` . We should make this configurable . 
` incase of incremental consumer` -> only if it is part of incoming dataset .

Reply:
can we file a follow up task for making it configurable.
btw, I don't understand your response on incremental consumer question. can u help clarify.

Reply:
`_corrupt_record ` this column is intermediate state , it's never commited to base table as column . before commiting error table , we separate out valid / error events and drop this column . 
https://issues.apache.org/jira/browse/HUDI-6181

Reply:
So is my understanding correct that this column is never written to storage irrespective of whether it is an incremental source or something else? It is only being used for internal error table validation but never written.

Reply:
yes